wp-content/mu-plugins/register-rest-routes.php

```diff
@@ -288,3 +290,141 @@
     }
   ));
 });
+
+add_action('rest_api_init', function() {
+  $v = 'api/v1'; // namespace for the current version of the API
+
+  $exp = WEEK_IN_SECONDS; // expiration of the transient caches  
+
+  function LogEventData($eventData){
```

Comment:
- [ ] Use separate functions for logging event data and transforming event data. This function should not return anything, and you should use a separate function that has `return array('event'=>$event_type, 'email'=>$email, 'templateName'=>$templateName)`. 
- [ ] (minor) use snake case instead of camel case: `log_event_data($event_data)` instead of `LogEventData($eventData)`